src/main/java/com/example/demo/modules/debt/DebtService.java

```diff
@@ -14,6 +13,6 @@
 
     List<DebtResponse> findAllDebt();
     DebtResponse createDebt(CreateDebt request) throws NotFoundException;
-    Debt proposeDebt(Debt oldDebt, Share stock, Timestamp timestamp, User user);
-    DebtResponse acceptDebt(long debtId, long userId) throws NotFoundException;
+    void proposeDebt (User user, ProposeDebt proposeDebt) throws Exception;
```

Comment:
einen ganzen User mitzugeben ist schlecht da die json dann wieder unendlich lang wird. eventuell nur username oder auch mittels des token ?

Reply:
Da muss forntend keinen Benutzer mitgeben, das wird automatisch durch token gemacht, das ist die Service Klasse, schau in Controller rein